### ISSUE TITLE
Create useLerp hook

### ISSUE THREAD

**Issue by VictorGa** [mediamonks/react-kit#241]
Lerping hook for simple transitions. Properties would be initial number, final number, duration, ease method.

**Comment by ReneDrie** [comment]
The `lerp` as a function is already available (gsap.utils.interpolate). But I assume this does the animationFrame loop, and keeps updating the current value by interpolating to the target value.

**Comment by VictorGa** [comment]
Yes, exactly that. I didn't mean the formula in a hook. Mostly a way to interpolate the values in an easier way using a hook for it, regardless if we use interpolate from gsap or a formula, could be handy. We have an example from @evertmonk. I'll create a PR for it soon.
